module/qr_code.py

```diff
@@ -1,5 +1,162 @@
 """ Provides a class in which to save QR code data."""
 
+from math import sqrt
+
+
+class Coordinate:
+    """ This class is used to represent x and y values of a coordiante. """
+
+    def __init__(self, x=0, y=0):
+        """ The constructor.
+        :param x: X value of the coordiante. 
+        :param y: Y value of the coordiante.
+        :return: Nothing.
+        """
+        self.x = x
+        self.y = y
+
+    def __repr__(self):
+        """ Represents the coordinate
+        :return: Returns a string with the x and y values.
+        """
+        return "{}, {}".format(repr(self.x), repr(self.y))
```

Comment:
Would (x, y) be a more logical representation? An example would be (0, 4) instead of 0, 4

Reply:
Fixed on line 22 in commit https://github.com/R2D2-2019/vision/pull/11/commits/2d4a20c0fd93867a7d668319a7c45ef5a00d22f7